src/script/event/EventService.ts

```diff
@@ -127,18 +127,30 @@ export class EventService {
 
     try {
       if (this.storageService.db) {
```

Comment:
in which case do we not have a `db`? Did you find a reason why we have those 2 branches?

Reply:
i could never get that branch to go off, not sure tbh

Reply:
oohhh it those experiment where we have different DB engines under the hood 😠 
![image](https://user-images.githubusercontent.com/1090716/212969532-faebc274-7766-45f6-8594-da943ccec29d.png)

Reply:
I guess we need to handle both scenarios 😬

Reply:
should we remove it?

Reply:
okay then